app/models/branches_of_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'common/models/base'
+
+# Cemetery model
+class BranchesOfService < Common::Base
+  include ActiveModel::Validations
+
+  # Cemetery numbers are 3-digits, implying < 1000 in total - we want all of them for populating application forms
+  per_page 1000
+  max_per_page 1000
+
+  validates :code, :begin_date, :end_date, :flat_full_descr,
+            :full_descr, :short_descr, :state_required, :upright_full_descr, presence: true
```

Comment:
Do we need all of these validations? As I understand it, Cemetery is a get request, so we don't need to validate the GET request. However, if you're also using this same object for building the form, I can understand the need for validation, but doesn't the application just require sending a cemetery code? Why all of the additional validation?